src/Writer/XlsExcelWriter.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+/**
+ * .xls writer bridge for library `phpoffice/phpexcel`
+ * @see https://github.com/PHPOffice/PHPExcel
+ *
+ * @package Exporter\Writer
+ * @author Lukáš Brzák <lukas.brzak@email.cz>
+ */
+class XlsExcelWriter implements TypedWriterInterface
```

Comment:
See [this article](https://ocramius.github.io/blog/when-to-declare-classes-final/). TL;DR : in order to get PRs that improve this class to make it more flexible, and encourage a better design (composition as opposed to inheritance)